k8s/namespaces/reform-scan/reform-scan-notification-service/aat.yaml

```
@@ -15,19 +14,8 @@ metadata:
     repository.fluxcd.io/functional: java.functionaltests.image
```

This might need changem pls have a look at other apps ```suggestion
    repository.fluxcd.io/functional: java.functionaltests.image
```